scripts/update_schemastore.py

```diff
@@ -0,0 +1,110 @@
+"""Update uv.json in schemastore.
+
+This script will clone astral-sh/schemastore, update the schema and push the changes
+to a new branch tagged with the uv git hash. You should see a URL to create the PR
+to schemastore in the CLI.
+"""
+
+from __future__ import annotations
+
+import json
+from pathlib import Path
+from subprocess import check_call, check_output
+from tempfile import TemporaryDirectory
+
+schemastore_fork = "git@github.com:astral-sh/schemastore.git"
+schemastore_upstream = "git@github.com:SchemaStore/schemastore.git"
+uv_repo = "https://github.com/astral-sh/uv"
+root = Path(
+    check_output(["git", "rev-parse", "--show-toplevel"], text=True).strip(),
```

Comment:
Minor preference not to run a subprocess outside of `main()`

Reply:
(This is copied over from Ruff, but I shall change!)